### DISCUSSION TITLE
Policy just to block downloads, possible?

### DISCUSSION THREAD

**Discussion by TabultaT** [mozilla/policy-templates#938] [Ideas]
I've tried to tinker with this template (thanks for this) to block downloads on firefox from downloading anything, by pointing the default directory to the admin account. I'm blocking other standard users of a mac computer. It worked and I used about:config to lock it down. However, drag and drop a download link to the download toolbar icon circumvents any settings and allows me to download wherever I want.

Is someone kind enough to tell me (or help create) if theres a way to completely block downloads in firefox? It's very easy with chrome. Only thing stopping my method from working is the pesky download toolbar icon that ignores all rules.

**Comment by mkaply** [comment]
What's the policy that Chrome uses to block all downloads?

**Reply by TabultaT** [reply]
DownloadsRestrictions in regedit and set to 3.

**Comment by mkaply** [comment]
I'll add this to our backlog, but I have no timeframe.

Even Google says this should be used on rare occasions.
